### ISSUE TITLE
Implement an analysis collection object

### ISSUE THREAD

**Issue by squirrelo** [biocore/qiita#857]
Similar to the idea of an investigation, which contains multiple studies, we need a collection object of some sort that contains many analysis objects. This will allow the many sub-analyses that go into one total analysis/paper to be easily referenced and found during final use for figure creation and paper writing. 

It would also be useful for this to have an "important" button for the analysis results in a collection, allowing you to highlight specific important results across the analyses in a collection object. For example, if you have 7 analyses in the object, and the beta diversity from analysis 3 is particularly important, you can highlight it like in gmail.

**Comment by josenavas** [comment]
That's confusing to me. I though that we will able to add more jobs to a given analyses once it is done. I'm missing something?

**Comment by squirrelo** [comment]
My understanding is that once a job is done running it's set in stone, but we can branch it using the analysis chain table. These two would then go into the collections object. There's also the case of subsetting your data and running another analysis, which would also end up as two separate analyses in our current setup, but still be in one overarching project. This just makes it easier for users to bookkeep their projects as they run more exploratory analyses.

**Comment by josenavas** [comment]
Thanks. Agree re: subsetting. Disagree on setting it in stone. If the data is the same, we should be able to add a job. Eg: if we have a meta-analysis with taxa summaries and we want to run beta diversity, we should be able to add that job to the meta-analysis rather than branching.

**Comment by squirrelo** [comment]
That makes sense in terms of not needing branching. So both setups need to be applied to the UI in this case: Adding more jobs to an analysis and having a collections object.

**Comment by josenavas** [comment]
Yes. In terms of expanding the discussion. Which will be the workflow? The user creates the analysis group and then the analysis or you create the analysis as it is currently and then you can later add them to an analysis group? I like the later...

**Comment by adamrp** [comment]
I like the latter as well.

**Comment by squirrelo** [comment]
Both would be best. You can either create some an analyses, realize you need a collection object, then create one and add them to it. Alternately you have already done several analyses and created a collection for them, so when you create another analysis there should be an easy way to automatically add that new analysis to the existing collection.

**Comment by josenavas** [comment]
Ok, but the creation of the analysis group should not be required. I.e. when you are creating an analysis, you should be able to choose an analysis group, but not required. This will also require changes on the analysis list page, so we have a list of available analysis groups and analysis themselves.

**Comment by squirrelo** [comment]
Yup, agree with all of that.

**Comment by antgonza** [comment]
Collection was completely deprecated in the last analysis refactor, thus closing.
